documentation/api/query/v4/pql.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "PuppetDB 5.2 » Puppet query language (PQL) » Reference guide"
+title: "PQL Reference guide"
 layout: default
```

Comment:
I'm somewhat tempted to keep the full name here since, if nothing else, it looks like that's what shows up in the search engine match line, and so that one line helps whether you know the full name or the acryonym, but I don't feel strongly about it. i.e. "Puppet query language (PQL) Reference guide".